tests/integration_tests/sink_hang/run.sh

```diff
@@ -42,7 +42,7 @@ function run() {
 	*) SINK_URI="mysql://normal:123456@127.0.0.1:3306/?max-txn-row=1" ;;
 	esac
 
-	export GO_FAILPOINTS='github.com/pingcap/tiflow/cdc/sink/MySQLSinkHangLongTime=1*return(true);github.com/pingcap/tiflow/cdc/sink/MySQLSinkExecDMLError=9*return(true)'
+	export GO_FAILPOINTS='github.com/pingcap/ticdc/cdc/sink/MySQLSinkExecDMLError=2*return(true)'
```

Comment:
We have rename to tiflow in release branch too.

Reply:
fixed